content/docs/learning-guide/tutorials/remote-repositories/create-remote-repository.md

```diff
@@ -0,0 +1,193 @@
+
+<ul><li>For example: Gems should go inside Gems folder, projects should be placed into Projects folder, and project templates should go inside Templates folder.</ul><br>
+Note: Remember to push your local changes to your version control remote repository
+</ul>
+
+
+<br>
+
+## Create Your Own Gem For Your Project (Optional)
+---
+You can enjoy the freedom of creating your own gem and upload it with your remote repo.<br>
+Please look at next section **Create release archive** if you want to make your gem avaliable to the world! <br>
+
+Follow the below command to call the `--create-gem` and pass in the path to where you want your gem to be stored (If you don't specify a name for your gem, O3DE will use the folder name you passed in, which is the last parameter of your provided path).
+<ul><ul><li>For example: "C:\o3de_Projects\RemoteProjectName\Gems\MyGem" Your gem name will be "MyGem" and its located in the Gems folder that was created during create-repo step</ul>
+<br>
+Note: Always put your gems in the Gems folder to avoid later confusion, Please follow the below template and pass your own gem in the path inside the Gems folder.
+
+```
+o3de.bat create-gem --gem-path "C:\o3de_Projects\<RemoteProjectName>\Gems\MyGem" --gem-name "SuperGem"
+```
+<ul>
+
+This step registers your gem to the O3DE engine, you will be able to find your gem under the Gems tab in your project managaer. Be sure to use the `--edit-repo-properties` `--add-gem` in order to add your gem to your project in order to work with your gem.<br>
+<ul> 
+For example: Add your gem to your project when working locally:
+<ul>
+
+```
+o3de.bat edit-repo-properties --repo-path "C:\o3de_Projects\<RemoteProjectName>\repo.json" --add-gem "C:\o3de_Projects\<RemoteProjectName>\Gems\MyGem"
+```
+
+</ul>
+</ul>
+<br>
+
+However, pushing your changes to your version control repository would not allow other user to use your gem, move on to the next step if you want to create a downloadable `release archive` for your gem to be shared with the world. Make sure  you push all your changes before moving to the next step to share your gem with the world.
+</ul>
+</ul>
+
+<br>
+
+## Create Release Archive
+---
+The primary purpose of a **Release archive** is to provide a well-defined and stable version of the software that can be easily shared, installed, and distributed.<br> 
+
+In O3DE we provide the option for users to create a release archive with .zip format.<br>
+
+Due to user safty, we use `sha-256` to hash our files before creating the release archive. Thus, you won't be able to modify your `repo.json` file once your archive is created. Every time you make a change to your gem, including the repo.json file, the `sha-256` hash will change.<br>
+
+<ul>
+
+Note: You must know where your zip will be created online **download path** before you actually create the release archive zip.</ul>
```

Comment:
I don't think you need this note. You can explain this requirement when you describe the parameters to use with the edit-repo-properties command.